cli.go

```diff
@@ -340,6 +353,61 @@ func (i *CLI) setupMetrics() {
 	}
 }
 
+func loadStackdriverTraceJSON(ctx gocontext.Context, stackdriverTraceAccountJSON string) (*google.Credentials, error) {
+	credBytes, err := loadBytes(stackdriverTraceAccountJSON)
+	if err != nil {
+		return nil, err
+	}
+
+	creds, err := google.CredentialsFromJSON(ctx, credBytes, googlecloudtrace.ScopeTraceAppend)
+	if err != nil {
+		return nil, err
+	}
+	return creds, nil
+}
+
+func loadBytes(filenameOrJSON string) ([]byte, error) {
+	var (
+		bytes []byte
+		err   error
+	)
+
+	if strings.HasPrefix(strings.TrimSpace(filenameOrJSON), "{") {
+		bytes = []byte(filenameOrJSON)
+	} else {
+		bytes, err = ioutil.ReadFile(filenameOrJSON)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return bytes, nil
+}
+
+func (i *CLI) setupOpenCensus(stackdriverTraceAccountJSON string) error {
+	creds, err := loadStackdriverTraceJSON(gocontext.TODO(), stackdriverTraceAccountJSON)
+	if err != nil {
+		return err
+	}
+	sd, err := stackdriver.NewExporter(stackdriver.Options{
+		ProjectID: os.Getenv("TRAVIS_WORKER_GCE_PROJECT_ID"),
+		TraceClientOptions: []option.ClientOption{
+			option.WithCredentials(creds),
+		},
+		MonitoringClientOptions: []option.ClientOption{
+			option.WithCredentials(creds),
+		},
+	})
+	if err != nil {
+		return err
+	}
+	defer sd.Flush()
+	// Register/enable the trace exporter
+	trace.RegisterExporter(sd)
+	trace.ApplyConfig(trace.Config{DefaultSampler: trace.ProbabilitySampler(1.0)})
```

Comment:
I will remember to make this configurable since we're seeing tracing in Staging.